header-moh.php

```diff
@@ -4,11 +4,11 @@
  *
  * Displays all of the <head> section and everything up till div#breadcrumb
  *
- * @package WordPress
- * @subpackage Twenty_Twelve
- * @since Twenty Twelve 1.0
+ * @package music_oral_history
+ * @since 2.0.0
```

Comment:
This tag is tricky - I know some places name `Twenty_Twelve` as their value for this, but the way I understand the intent of the tag that's the wrong way to go about it (and at any rate, header-moh doesn't come from Twenty_Twelve).

I don't know that we could identify when exactly this template was added - so I guess just naming it as 2.0.0 is fine?